.github/workflows/tests.yml

```
@@ -27,3 +28,28 @@ jobs:
     - run: npm run prettier # ensure code is formatted
     - run: npm run build # ensure it can build
     - run: npm test # ensure tests pass
```

If you want, you could fold the test-webcrypto command into this job like this:

```suggestion
    - run: npm test # ensure tests pass
    - if: matrix.package == 'crypto' && matrix.node-version == '18.x'
      run: npm run test-webcrypto # ensure webcrypto tests pass
```

But what you have currently is fine too.

---

I don't have much of a preference. Lean slightly more towards the existing pattern as it's then listed as an explicit step in the pr build list. Can change if you feel strongly.

That said, I do think we want to make use of these conditionals to only run tests when code in the appropriate package is changed. Will leave that for follow up prs!

---

Either is fine!